native/cocos/renderer/gfx-base/GFXDescriptorSet.h

```diff
@@ -42,8 +42,10 @@ class CC_DLL DescriptorSet : public GFXObject, public RefCounted {
     virtual void forceUpdate() = 0;
 
     virtual void bindBuffer(uint32_t binding, Buffer *buffer, uint32_t index);
-    virtual void bindTexture(uint32_t binding, Texture *texture, uint32_t index);
     virtual void bindSampler(uint32_t binding, Sampler *sampler, uint32_t index);
+    virtual void bindTexture(uint32_t binding, Texture *texture, uint32_t index, AccessFlags flags);
+
+    void bindTexture(uint32_t binding, Texture *texture, uint32_t index);
```

Comment:
Have we inlined bindTexture in the header?

Reply:
`bindTexture` implemented in GFXDescriptorSet.cpp